### PR TITLE
Update attachment_adobe_image_lure.yml

### PR DESCRIPTION
**Pull request by morriscode**
FN coverage. Adding message screenshot

### CODE REVIEW

**Comment by morriscode** [comment]
Testing looks great, this is gtg.

File: detection-rules/attachment_adobe_image_lure.yml [hunk 1]
```diff
@@ -9,25 +9,26 @@ source: |
...
-                  )
-                  and (
-                    (length(body.links) > 0)
-                    or (length(.scan.url.urls) > 0 or length(.scan.pdf.urls) > 0)
-                  )
-          )
+  and (
+    any(file.explode(beta.message_screenshot()),
+        any(ml.logo_detect(beta.message_screenshot()).brands, .name == "Adobe")
+        and 0 < length(body.links) < 10
```
Comment (jkamdjou) [inline detection-rules/attachment_adobe_image_lure.yml:15]: this feels a little broad -- basically flagging on a message with an Adobe logo + a link? am i understanding correctly?

Comment (morriscode) [reply detection-rules/attachment_adobe_image_lure.yml:15]: and there's no display text on the links.
It seems broad, but it's fired a collective 180 times in testing, and there's only 4 hits with no other rule overlap. 

Of those 180 firings, 73 of them have actions confirming TP, (quarantine, trash, webhook) with 0 actions indicating FP's.

**Review by aidenmitchell — APPROVED** [review]

**Review by aidenmitchell — APPROVED** [review]

**Pull request merged**